cg/meta/archive/archive.py

```diff
@@ -53,7 +55,29 @@ def __init__(
 
     def archive_files(self, files: List[FileAndSample], archive_location: ArchiveLocations) -> int:
         archive_handler: ArchiveHandler = ARCHIVE_HANDLERS[archive_location](self.data_flow_config)
-        return archive_handler.archive_folders(files_and_samples=files)
+        return archive_handler.archive_files(files_and_samples=files)
+
+    def archive_all_non_archived_spring_files(
```

Comment:
Consider splitting this functions in several smaller ones, maybe something like

```python
    def archive_all_non_archived_spring_files(
        self, spring_file_count_limit: Optional[int] = None
    ) -> None:
        """Archives all non archived spring files."""

        files_to_archive = self._fetch_non_archived_files(spring_file_count_limit)
        files_and_samples = self._associate_files_with_samples(files_to_archive)

        for archive_location in ArchiveLocations:
            selected_files = self._filter_files_by_location(files_and_samples, archive_location)
            self._archive_selected_files(selected_files, archive_location)

```

Reply:
I think this is easier to read at least since it is shorter and encapsulates some of the logic in functions with clear names 👍

Reply:
Or maybe this is how it is now already, its just polluted with type hints and named arguments that make it hard to read 😅

Reply:
I think the for loop body can be extracted to a single function if you feel like it

Reply:
Haha the suggestion is basically how it is at the moment, but I could definitely extract the for loop :)

Reply:
And unfortunately, without the named arguments, the line would still be split since it exceeds the black threshold by one (1) character 😞